### DISCUSSION TITLE
I need clarification on the Modelable behavior.

### DISCUSSION THREAD

**Discussion by zircuitz** [livewire/livewire#8979] [1. Help]
Can I ask for a clarification on the Modelable behaviour?

If I update a property on a child component that has been set to modelable from it's parent: Does updating it from the child component, also update the property on the parent? Or do I still have to use events for such operations?

I think this should be clarified in the documentation. Especially if it doesn't work like that.

I've been having some issues regarding this. It doesn't say anything about that specifically in the documentation, but I just assumed it would work like that. After some testing, it doesn't seem like that's the case.

What happens when I test is that the parent and child component gets de-synced. So for example: If I change a property boolean to false on the child component: It doesn't get updated to false on the parent component. Rather: The property gets updated to false on the child component, but the parent property remains true. And If I update something on the parent component that causes it to re-render: It will change the child's property back to true. Causing unexpected behavior.

If I update from the parent component: It always gets updated on the child component as well. Working as expected.

It seems this issue has been [brought up before](https://github.com/livewire/livewire/discussions/6114), but the issue has gone stale and unanswered.

**Comment by joshhanley** [comment]
@zircuitz Are you using `wire:model.live`? If not then the parent component state won't update until it makes a request.

**Reply by zircuitz** [reply]
I can confirm I do mark it with `wire:model.live`. It does not change the behavior.
